src/tasks/LargeFileUploadTask.ts

```diff
@@ -65,9 +68,10 @@ export interface LargeFileUploadSession {
  * @property {number} size - Specifies size of the file
  */
 export interface FileObject {
-	content: ArrayBuffer | File;
+	content: any;
```

Comment:
make it generic instead of having any. Having any will require consumers to do a bunch of type testing/casting.

Reply:
Done